go.mod

```diff
@@ -55,6 +54,8 @@ require (
 	resenje.org/web v0.4.3
 )
 
+require golang.org/x/exp v0.0.0-20230725012225-302865e7556b // indirect
```

Comment:
Why is this defined separately?

Reply:
it's being pulled out automatically for some reason by the tool chain, I moved it back in place manually